src/main/java/bot/commands/audio/utils/VoiceChannelUtils.java

```diff
@@ -110,68 +97,69 @@ public static void SearchAndPlaySong(JDA jda, String argument, String guildId, S
             argument = "ytsearch: ".concat(argument);
         }
-
+        playerManager.loadItem(argument, new AudioSearchResultHandler(trackScheduler, audioPlayerSendHandler, channel
+                , argument, playTop, userInfo));
     }
 
-    private static boolean isValidURL(String urlString){
-
+    private static boolean isValidURL(String urlString) {
         try {
             URL url = new URL(urlString);
             url.toURI();
             return true;
-        }catch (Exception exception){
+        } catch(Exception exception) {
             return false;
         }
     }
-    public static void setPauseStatusOnAudioPlayer(Guild guild, MessageChannel channel,
-                                                   Member member, boolean pauseStatus) throws IllegalArgumentException,
-            IllegalAccessException {
-
-        AudioManager audioManager = guild.getAudioManager();
-
-        if (!audioManager.isConnected()){
-            channel.sendMessage(TextChannelResponses.BOT_NOT_CONNECTED_TO_VOICE).queue();
-            throw new IllegalAccessException("Bot not connected to the voice channel");
-        }
-
-        if (audioManager.getConnectedChannel() != null && !audioManager.getConnectedChannel().getMembers().contains(member)){
-            channel.sendMessage(TextChannelResponses.NOT_CONNECTED_TO_VOICE_MESSAGE).queue();
-            throw new IllegalAccessException("Member not in the voice channel");
-        }
-        AudioPlayerSendHandler audioPlayerSendHandler = (AudioPlayerSendHandler) audioManager.getSendingHandler();
-        AudioPlayer audioPlayer = audioPlayerSendHandler.getAudioPlayer();
-        if (audioPlayer.isPaused() == pauseStatus){
-            throw new IllegalArgumentException("Setting the same pause status");
-        }
-        audioPlayer.setPaused(pauseStatus);
-
-    }
 
-    public static AudioPlayerSendHandler getAudioPlayerSendHandler(JDA jda, String guildId){
-
-        if (guildId == null || guildId.equals("")){
+    public static AudioPlayerSendHandler getAudioPlayerSendHandler(JDA jda, String guildId) throws IllegalArgumentException {
+        if (guildId == null || guildId.equals("")) {
             throw new IllegalArgumentException("Guild ID is NULL");
-
         }
 
         Guild guild = jda.getGuildById(guildId);
-
-        if (guild == null){
+        if (guild == null) {
             throw new IllegalArgumentException("Guild is NULL is the ID correct?");
         }
 
         AudioManager audioManager = guild.getAudioManager();
 
-        if (!audioManager.isConnected()){
+        if (!audioManager.isConnected()) {
             throw new IllegalArgumentException("Not currently connected to the voice channel");
         }
 
         return (AudioPlayerSendHandler) audioManager.getSendingHandler();
     }
 
-}
+    public static void setPauseStatusOnAudioPlayer(CommandEvent event, boolean pauseStatus) throws IllegalArgumentException,
```

Comment:
This static method returns some information about the audio player in addition to the status